docs/concepts/sensor-models/sensor-models.md

```diff
@@ -0,0 +1,90 @@
+
+!!! INFO "Reference System vs Reference Frame"
+    A __reference system__ provides an origin, axes / planes, and a fixed point that allows for the description of an object.  A __reference frame__ is a description within the context of that reference system, and provides the location of an object at a given moment in time.
+
+Finally, reference frames often have dependencies on other reference frames, which results in a "frame chain" that must be calculated. For example, consider a common scenario in which a camera is intended to record an image of a planetary body.  The camera is mounted on a gimbal, which is mounted on the spacecraft.  In order to move from the image plane to a BCBF system, it is necessary to know the location of the camera and the planetary body.  However, in order to know the location of the camera, it is necessary to know the extrinsics not only of the camera, but also of the gimbal and the spacecraft to which it is mounted.  The dependency chain that is formed is known as a "frame chain," and the entire chain of extrinsics must be calculated in order to accurately model the objects' relative locations.
+
+
+### Sensor Models
+Briefly, a sensor model encapsulates a sensor's geometric information. In this context, "geometric information" refers to two classes of information -- interior (intrinsic) and exterior (extrinsic).  Intrinsic information describes information that is internal to the sensor, such as focal length,  resolution, distortion, etc. Extrinsic information relates to the sensor's physical position, velocity, and orientation, which is to say that extrinsic information may be used answer the question "where is the sensor and where is it pointing."
+
+@TODO expand this section, include rigorous and rational models, uses of sensor models 
+
+!!! NOTE
+    The overarching goal of a sensor model is to provide a means for moving between and among a variety of reference frames, which is a necessary step in facilitating geolocation and other geodetic processes.
+
+
+## The Community Sensor Model
+
+Because the only requirement of a sensor model is that it models a sensor's geometric information, the design and implementation of a sensor model largely remains open-ended.  Of particular importance is the sensor's interface, i.e. what information is required to create the model and what information is produced by the model.  Without a concrete interface specification, each sensor model may require and provide its own set of information, which makes it incredibly difficult to facilitate interoperability between sensor models. In order to address these concerns about interoperability, the US defense and intelligence community established the Community Sensor Model (CSM) working group, which is responsible for developing and maintaining a set of standards upon which sensor models can be implemented.  The resulting CSM standard describes an application programming interface (API) for multiple sensor types.
```

Comment:
Here is a nice, concise definition: http://defs.opengis.net/vocprez/object?uri=http://www.opengis.net/def/sensor-model/csmwg/csm

Then important part is that this is a mathematical model.

Reply:
Link to the CSM? https://gwg.nga.mil/gwg/focus-groups/Community_Sensor_Model_Working_Group_(CSMWG).html#:~:text=The%20Community%20Sensor%20Model%20(CSM,interoperable%20generic%20physical%20sensor%20models.